applets/clock/clock.c

```diff
@@ -987,6 +995,78 @@ position_calendar_popup (ClockData *cd)
 
         gtk_window_move (GTK_WINDOW (cd->calendar_popup), x, y);
         gtk_window_set_gravity (GTK_WINDOW (cd->calendar_popup), gravity);
+    }
+#endif
+
+    /*Only build wayland support when building in process*/
+#if defined(CLOCK_INPROCESS) && defined(HAVE_WAYLAND) && defined(GDK_WINDOWING_WAYLAND)
+    if (GDK_IS_WAYLAND_DISPLAY (gdk_display_get_default ()))
+    {
+        GtkWindow *window;
+        GdkWindow *panelwin;
+        GtkWidget *toplevel;
+        int  x, y, w, h, panel_w, panel_h;
+
+        /*Get the calendar window dimensions*/
+        window = (GTK_WINDOW (cd->calendar_popup));
+        gtk_window_get_size (window, &w, &h);
+        /*Find the position of the applet*/
+        gdk_window_get_origin (gtk_widget_get_window (cd->panel_button),
+                                                                      &x, &y);
```

Comment:
style: alignment

Reply:
Moved that code back to line up with the first letter in the code inside the opening parenthesis, as is common elsewhere in the file